### PR TITLE
Code cleanup by using .format

### PR DESCRIPTION
**Pull request by edawine**
Checked Python codes by using flake8.
Changed % string format into .format for better readibility.

### CODE REVIEW

**Comment by andrewschoen** [comment]
ok to test

**Comment by alfredodeza** [comment]
I am not sure I see the value on changing existing code just to use `.format()` 

`.format()` is useful when there are named items (I think you managed to get one case of those). Blanket style changes like this one aren't something we are pursuing at the moment.

Thanks for taking the time to get this out though.

**Comment by edawine** [comment]
Okay, thank you for the feedback.

Can I edit my PR so it only changes the case you mentioned (Is it the one with '%s/%s...'?) or not?

File: chacra/controllers/util.py [hunk 1]
```diff
@@ -36,13 +36,13 @@ def relative(self):
...
 
     def __str__(self):
-        return "{0}{1}{2}{3}{4}{5}".format(
-            self.years,
-            self.months,
-            self.days,
-            self.hours,
-            self.minutes,
-            self.seconds,
+        return "{years}{months}{days}{hours}{minutes}{seconds}".format(
```
Comment (alfredodeza) [inline chacra/controllers/util.py]: this is one that looks OK to me

File: chacra/models/repos.py [hunk 2]
```diff
@@ -43,12 +43,12 @@ def __init__(self, project, ref, distro, distro_version, **kwargs):
 
     def __repr__(self):
         try:
-            return "<Repo {}/{}/{}/{}/{}>".format(
-                self.project.name,
-                self.ref,
-                self.sha1,
-                self.distro,
-                self.distro_version,
+            return "<Repo {name}/{ref}/{sha1}/{distro}/{version}>".format(
```
Comment (alfredodeza) [inline chacra/models/repos.py]: this one too

File: chacra/models/repos.py [hunk 3]
```diff
@@ -75,19 +75,19 @@ def __json__(self):
...
     @property
     def uri(self):
-        return "{}/{}/{}/{}/{}/flavors/{}/".format(
-            self.project.name,
-            self.ref,
-            self.sha1,
-            self.distro,
-            self.distro_version,
-            self.flavor,
+        return "{name}/{ref}/{sha1}/{distro}/{version}/flavors/{flavor}/".format(
```
Comment (alfredodeza) [inline chacra/models/repos.py]: this looks ok too

File: chacra/util.py [hunk 4]
```diff
@@ -47,12 +47,12 @@ def repo_paths(repo):
     paths = {}
 
     # e.g. ceph-deploy/master/ubuntu/trusty
-    paths['relative'] = '%s/%s/%s/%s/flavors/%s' % (
-        repo.ref,
-        repo.sha1,
-        repo.distro,
-        repo.distro_version,
-        repo.flavor
+    paths['relative'] = '{ref}/{sha1}/{distro}/{version}/flavors/{flavor}'.format(
```
Comment (alfredodeza) [inline chacra/util.py]: this look ok

**Comment by alfredodeza** [comment]
This looks OK. Can you squash the revert commit and have one commit per file changed with a description of what is the commit about? (vs. 'Only change to .format as requested')

**Comment by edawine** [comment]
Done. Is there anything else I can do?

**Comment by alfredodeza** [comment]
Looks like you didn't get rid of 1d0416a2c185f6e24fc0db2b59120ba46cc60071

**Comment by edawine** [comment]
Is it OK now?

**Comment by alfredodeza** [comment]
Yep, thank you!

**Pull request merged**